### PR TITLE
[CI] fix: Mesheryctl lint checks failing test

### LINKED ISSUE #10631: [mesheryctl]: lint checks failing
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior
Several lint checks are failing in the recent GitHub Actions run for mesheryctl. This issue is linked to the following GitHub Actions run: [link to the GitHub Actions run](https://github.com/meshery/meshery/actions/runs/8466661227/job/23195915176).
image

### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by singh1203**
**Notes for Reviewers**

Added missing `.golden` after Consolidation of Mesheryctl app cmd to pattern cmd, and also upgraded download-artifact  `v3` -> `v4`

This PR fixes #10631 

Screenshot from 2024-03-30 00-56-41

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/test_adaptersv2.yaml [hunk 1]
```diff
@@ -164,7 +164,7 @@ jobs:
           config='{"contexts":{"local":{"endpoint":'$svcip',"token":"default","platform":"kubernetes","adapters":[],"channel":"stable","version":"latest"}},"current-context":"local","tokens":[{"location":"auth.json","name":"default"}]}'
           echo $config  > ~/.meshery/config.yaml
       - name: Download patternfile
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```
Comment (leecalcote) [inline .github/workflows/test_adaptersv2.yaml:167]: good

**Review by leecalcote — APPROVED** [review]
This is an improvement. Thank you, @singh1203

**Review by RipulHandoo — APPROVED** [review]
LGTM.

**Comment by vishalvivekm** [comment]
@singh1203 Let's discuss this in Meshery dev call today. I'm adding this as an agenda item to the meeting[ minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#).

**Comment by RipulHandoo** [comment]
@MUzairS15 This PR is good to go. 🚀

**Review by lekaf974 — APPROVED** [review]

**Comment by lekaf974** [comment]
LGTM

**Comment by leecalcote** [comment]
Thank you, gents! 💯

**Pull request merged**